conda_forge_tick/migrators.xsh

```diff
@@ -738,18 +738,9 @@ class Rebuild(Migrator):
 
 
 class CompilerRebuild(Rebuild):
-    bump_number = 1000
     migrator_version = 1
```

Comment:
Should you bump the migrator version?

Reply:
I'm not certain, I think if we do that it will invalidate the current cache of PRs it has issued causing it to PR against everything again.

Reply:
This is because we don't check if the current compiler is gcc7 (I'm not certain if there is a way for us to access that)

Reply:
Ahhh ok!